subprojects/test-kit/src/integTest/groovy/org/gradle/testkit/runner/GradleRunnerPluginClasspathInjectionIntegrationTest.groovy

```diff
@@ -37,9 +35,8 @@ import static org.hamcrest.CoreMatchers.containsString
 @InjectsPluginClasspath
 @InspectsBuildOutput
 @UsesNativeServices
-@SuppressWarnings('IntegrationTestFixtures')
+@SuppressWarnings('IntegrationTestFixtures') // result.output.contains does mean something different here
```

Comment:
> result.output.contains does mean something different here

What does this mean? BTW, it seems no effect to remove `@SuppressWarnings('IntegrationTestFixtures')` on my machine?

Reply:
We have a custom codenarc rule which suggests replacing result.output.contains with outputContains. If you remove the annotation, then codenarc will fail.

Reply:
Good to know, thanks for the explanation!